builder/core/data.py

```diff
@@ -550,7 +550,7 @@ class PKG_TOOLS(Enum):
     },
     'gcc': {
         'hosts': ['linux', 'manylinux', 'al2012', 'al2', 'freebsd'],
-        'targets': ['linux', 'freebsd'],
+        'targets': ['linux', 'freebsd', 'android'],
```

Comment:
note: there was a bug in old versions where it would always use default compiler targets to determine whether cross-compile is supported (and default specified all possible targets as supported). that bug was fixed, but data was not updated to indicate that gcc can cross compile to andorid